### PR TITLE
Support SSL config

### PR DESCRIPTION
**Pull request by sergluka**
Add Kafka SSL configuration;

### CODE REVIEW

File: skafka/src/main/scala/com/evolutiongaming/skafka/consumer/ConsumerConfig.scala [hunk 1]
```diff
@@ -112,6 +113,7 @@ object ConsumerConfig {
     excludeInternalTopics: Boolean,
     isolationLevel: IsolationLevel,
     saslSupport: SaslSupportConfig,
+    sslSupport: SslSupportConfig,
```
Comment (t3hnar) [inline skafka/src/main/scala/com/evolutiongaming/skafka/consumer/ConsumerConfig.scala:116]: could you please remove this line for sake of binary backward compatibility ?

File: skafka/src/main/scala/com/evolutiongaming/skafka/consumer/ConsumerConfig.scala [hunk 2]
```diff
@@ -31,6 +31,7 @@ final case class ConsumerConfig(
   excludeInternalTopics: Boolean             = true,
   isolationLevel: IsolationLevel             = IsolationLevel.ReadUncommitted,
   saslSupport: SaslSupportConfig             = SaslSupportConfig.Default,
+  sslSupport: SslSupportConfig               = SslSupportConfig.Default,
```
Comment (t3hnar) [inline skafka/src/main/scala/com/evolutiongaming/skafka/consumer/ConsumerConfig.scala:34]: this is not backward compatible change, could you please use same approach as here?
https://github.com/evolution-gaming/scassandra/blob/master/scassandra/src/main/scala/com/evolutiongaming/scassandra/CassandraConfig.scala#L37

File: skafka/src/main/scala/com/evolutiongaming/skafka/producer/ProducerConfig.scala [hunk 3]
```diff
@@ -29,6 +29,7 @@ final case class ProducerConfig(
   transactionTimeout: FiniteDuration                = 1.minute,
   transactionalId: Option[String]                   = None,
   saslSupport: SaslSupportConfig                    = SaslSupportConfig.Default,
+  sslSupport: SslSupportConfig                      = SslSupportConfig.Default,
```
Comment (t3hnar) [inline skafka/src/main/scala/com/evolutiongaming/skafka/producer/ProducerConfig.scala:32]: this is not backward compatible change, could you please use same approach as here?
https://github.com/evolution-gaming/scassandra/blob/master/scassandra/src/main/scala/com/evolutiongaming/scassandra/CassandraConfig.scala#L37

File: skafka/src/test/scala/com/evolutiongaming/skafka/consumer/ConsumerConfigSpec.scala [hunk 4]
```diff
@@ -45,6 +45,18 @@ class ConsumerConfigSpec extends AnyFunSuite with Matchers {
       clientCallbackHandlerClass      = Some(classOf[ConsumerConfigSpec]),
       loginCallbackHandlerClass       = Some(classOf[ConsumerConfigSpec]),
       loginClass                      = Some(classOf[ConsumerConfigSpec]),
+    ),
+    sslSupport = new SslSupportConfig(
```
Comment (t3hnar) [inline skafka/src/test/scala/com/evolutiongaming/skafka/consumer/ConsumerConfigSpec.scala:49]: no need for `new`, simple `SslSupportConfig` should work just fine,
same in other places

**Review by t3hnar — CHANGES_REQUESTED** [review]

File: skafka/src/test/scala/com/evolutiongaming/skafka/consumer/ConsumerConfigSpec.scala (hunk 4 above)
Comment (sergluka) [reply skafka/src/test/scala/com/evolutiongaming/skafka/consumer/ConsumerConfigSpec.scala:49]: Fixed

**Review by t3hnar — APPROVED** [review]

**Comment by t3hnar** [comment]
released under `v11.8.0`

**Pull request merged**